src/WebService/Client.php

```diff
@@ -13,14 +13,14 @@
 use MaxMind\WebService\Client as WsClient;
 
 /**
- * This class provides a client API for all the GeoIP2 Precision web service
- * end points. The end points are Country, City, and Insights. Each end point
- * returns a different set of data about an IP address, with Country returning
- * the least data and Insights the most.
+ * This class provides a client API for all the GeoIP2 Precision web services.
+ * The services are Country, City, and Insights. Each service returns a
+ * different set of data about an IP address, with Country returning the
+ * least data and Insights the most.
  *
- * Each web service end point is represented by a different model class, and
- * these model classes in turn contain multiple Record classes. The record
- * classes have attributes which contain data about the IP address.
+ * Each web service is represented by a different model class, and these model
+ * classes in turn contain multiple Record classes. The record classes have
```

Comment:
Not sure. It probably was intended to refer to the namespace specifically, but I agree it is inconsistent. I'll fix it.